configs/config.novacustom_ns5x

```diff
@@ -1,17 +1,18 @@
 CONFIG_LOCALVERSION="v1.3.0"
-CONFIG_VENDOR_CLEVO=y
+CONFIG_VENDOR_NOVACUSTOM=y
+CONFIG_MAINBOARD_VERSION="2.2A"
 CONFIG_FMDFILE="src/mainboard/clevo/tgl-u/vboot-rwa.fmd"
-CONFIG_MAINBOARD_VENDOR="Notebook"
 # CONFIG_CONSOLE_SERIAL is not set
+# CONFIG_ONBOARD_VGA_IS_PRIMARY is not set
```

Comment:
Shouldn't onboard VGA be primary? One doesn;t boot laptop with dGPU turned on

Reply:
Fixed